scripts/devops_tasks/test_run_samples.py

```diff
@@ -21,7 +21,12 @@
 root_dir = os.path.abspath(os.path.join(os.path.abspath(__file__), "..", "..", ".."))
 
 IGNORED_SAMPLES = {
-    "azure-eventgrid": ["__init__.py"]
+    "azure-eventgrid": [
+        "__init__.py",
+        "consume_cloud_events_from_eventhub.py",
+        "consume_cloud_events_from_service_bus_queue.py",
+        "consume_cloud_events_from_storage_queue.py",
+        "cs2_publish_custom_events_to_a_domain_topic"]
```

Comment:
Will be fixed in this https://github.com/Azure/azure-sdk-for-python/issues/16042